packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
I can't tell if this is at all related to this PR, but do we really drop this `a` binding, so `a === 1` when the default is evaluated? Ideally this is a TDZ error, but we should at least make it come out `undefined`.